packages/suite/src/components/suite/Labeling/components/AddressLabeling.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as accountUtils from '@wallet-utils/accountUtils';
+import { useSelector } from '@suite-hooks';
+import { AccountLabeling } from './AccountLabeling';
+
+interface AddressLabelingProps {
+    address?: string | null;
+    knownOnly?: boolean;
+}
+
+export const AddressLabeling = ({ address, knownOnly }: AddressLabelingProps) => {
+    const accounts = useSelector(state => state.wallet.accounts);
+
+    if (!address) {
+        return null;
+    }
+
+    const found = accountUtils.findAccountsByAddress(address, accounts);
```

Comment:
found is really generic, try something like `addressAccounts` or `foundAccounts`

Reply:
done